### ISSUE TITLE
App isn´t working

### ISSUE THREAD

**Issue by chrisandrew-dev** [spikespaz/search-deflector#54]
# Bug Description

Worked after the initial installation but does not work after restarting and I´ve been unable to get t to work since. Start bar searches open in Edge (not Chrome) using Bing (not Google).

## How to Reproduce

Replace the following placeholders with correct information and delete this line.

**Browser**: Chrome

**Search Engine:** Google

**Steps to reproduce the behavior:**

1. Go to Windows Search bar
2. Type anything
3. Click See web results 

## Expected Behavior

I expected my search query to launch Chrome browser and perform a Google search.

## Environmental Details

| Windows Version | 1903 |
| --- | --- |
| Build Number | 18362.418 |
| Insiders Preview | No|

**Comment by spikespaz** [comment]
Do you use any system cleaning utilities like CCleaner or something from IObit? Maybe the registry keys were deleted. Try running the installer again.

**Comment by chrisandrew-dev** [comment]
No, nothing like that. I tried that already. :(

**Comment by spikespaz** [comment]
Try changing the protocol handler like here.

https://github.com/spikespaz/search-deflector/wiki/Troubleshooting#search-results-still-open-in-edge

**Comment by chrisandrew-dev** [comment]
Hi Jacob,

PC is a new build and I noticed a few other bugs so I rolled back to a
fresh Windows install and so far so good.

Not sure if this helps but on the first search when prompted with
"Which program to use to perform this action blah, blah", I checked "Always
use this program..." this time in addition to choosing Search Deflector.
Sometimes the most obvious solution is the one we miss.

Regards,

Chris.
